rmw_connext_cpp/include/rmw_connext_cpp/connext_static_subscriber_info.hpp

```diff
@@ -55,4 +67,138 @@ class ConnextSubscriberListener : public DDS::SubscriberListener
   std::atomic<std::size_t> current_count_;
 };
 
+/**
+ * Remap the specific RTI Connext DDS DataReader Status to a generic RMW status type.
+ *
+ * @param mask input status mask
+ * @param event
+ */
+inline rmw_ret_t ConnextStaticSubscriberInfo::get_status(
+  const DDS_StatusMask mask,
+  void * event)
+{
+  switch (mask) {
+    case DDS_StatusKind::DDS_SAMPLE_REJECTED_STATUS: {
```

Comment:
I think we need to focus only on the QoS events that we're actually fronting (deadline and liveliness) to the user and remove the conversion for all these others for now. Let's keep the implementation small and focused on our goal.